package.json

```diff
@@ -8,10 +8,16 @@
     "lint": "yarn run lint:js && yarn run lint:scss",
     "lint:js": "standardx 'app/assets/javascripts/**/*.js' 'spec/javascripts/**/*.js'",
     "lint:scss": "stylelint app/assets/stylesheets/",
-    "jasmine:prepare": "RAILS_ENV=test bundle exec rails assets:clobber assets:precompile",
-    "jasmine:ci": "yarn run jasmine:prepare && yarn run jasmine-browser-runner runSpecs",
+    "rails:precompile-assets": "RAILS_ENV=test bundle exec rails assets:clobber assets:precompile",
+    "nyc:instrument": "yarn run nyc instrument public/assets/frontend .nyc_output/assets",
+    "jasmine:prepare": "yarn run rails:precompile-assets && yarn run nyc:instrument",
+    "nyc:report": "yarn run nyc report",
+    "jasmine:ci": "yarn run jasmine:prepare && yarn run jasmine-browser-runner runSpecs && yarn run nyc:report",
     "jasmine:browser": "yarn run jasmine:prepare && yarn run jasmine-browser-runner"
   },
+  "imports": {
+    "#root/*": "./*"
```

Comment:
Depending on if the other changes are made in [nyc-test-coverage-reporter.mjs](https://github.com/alphagov/frontend/pull/4142/files#diff-cad9e4a78533bc33cf9c9b72531808b7be8503ad13528c114c664067d243893c) we may only need a reference to nyc_output, although I can see the benefit of having a reference to the root directory as well, is it possible to add a reference to the nyc_output folder here as well?

Another approach that we could use is creating a JS file that contains the paths to be used in the application, example from the govuk-frontend (Design System)
https://github.com/alphagov/govuk-frontend/blob/a50eb60c9583aea38e82625889fa7917a2f2fad9/shared/config/paths.js

Reply:
We can also add a specific path instead of using a pattern.

Reply:
After the discussion we decided to keep the imports solution. Nevertheless, because of getting rid of file removal from the reporter, the reporter can be simplified.

Reply:
Done